modules/common/src/main/java/com/opengamma/sdk/common/ServiceInvoker.java

```diff
@@ -43,6 +43,10 @@
  * such as via the try-with-resources statement.
  */
 public final class ServiceInvoker implements AutoCloseable {
+  /**
+   * The User Agent header to attach to all outgoing HTTP requests.
+   */
+  public static final String USER_AGENT = "OG-JAVA-SDK";
```

Comment:
Definitely think this is worthwhile but I think we can put more information into this string. At the very least, user agent strings normally begin `[Application name]/[Version]`. The application name is normally something quite friendly rather than looking like a programmatic constant.